### PR TITLE
fixing routing

### LINKED ISSUE #70: Fix routing

### PR DESCRIPTION
**Pull request by jas-haria**

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -14,15 +14,18 @@ import {
 } from "react-router-dom";
 import DeliveryStatus from './Components/DeliveryStatus';
 
-var permissions = ""
+// var permissions = ""
```
Comment (AugustusSaalfeldCMU) [inline frontend/src/App.tsx:17]: remove dead code.

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -14,15 +14,18 @@ import {
...
 
 function App() {
   return (
     <Router>
       <div>
-        <CustomNavbar permissions = {permissions}/>
+        {/* <CustomNavbar permissions = {permissions}/> */}
```
Comment (AugustusSaalfeldCMU) [inline frontend/src/App.tsx:23]: dead code

File: frontend/src/App.tsx [hunk 3]
```diff
@@ -14,15 +14,18 @@ import {
...
+        <CustomNavbar/>
       <Routes>
-          {permissions == "Employee" ? <Route path="/pending" element= {<PendingOrderCard/>} /> : <Route path="/checkout" element= {<Cart/>} />}
+          {/* {permissions == "Employee" ? <Route path="/pending" element= {<PendingOrderCard/>} /> : <Route path="/checkout" element= {<Cart/>} />} */}
```
Comment (AugustusSaalfeldCMU) [inline frontend/src/App.tsx:26]: dead

**Review by AugustusSaalfeldCMU — COMMENTED** [review]
Just some dead code to clean up, otherwise ready for merging.

**Comment by ncr0517** [comment]
Amazing routes, Jas

**Pull request merged**